### ISSUE TITLE
New 7TV V3 extension conflict

### ISSUE THREAD

**Issue by robmonte** [tmarenko/twitch_chat_antiban#9]
7TV has a new V3 extension which actually unloads the official Twitch webchat and replaces it entirely. This causes a conflict at the moment, where this extension reads the unloaded chat as a ban and starts up, but then the 7TV chat also starts and they overlap.

I think there may need to be an additional element detection added to this extension to actually wait for the text "You've been banned from chat" or something similar to prevent loading on top of the 7TV chat.

**Comment by tmarenko** [comment]
Can you provide a link to V3 extension? I've tested latest one from Chrome store and it works as expected

**Comment by robmonte** [comment]
Yeah, the extension is currently in Beta and hasn't fully replaced the current stable 7TV addon yet. It is located here: https://chrome.google.com/webstore/detail/7tv-beta/fphegifdehlodcepfkgofelcenelpedj

Active development here: https://github.com/SevenTV/ExtensionV3

**Comment by robmonte** [comment]
For what it's worth, the V3 extension has exited beta and is now in RC stage. It's set to replace the standard 7TV extension I think within a few weeks. The original extension repository (located here: https://github.com/SevenTV/Extension) has now swapped over to the V3 extension.

**Comment by tmarenko** [comment]
Done in https://github.com/tmarenko/twitch_chat_antiban/commit/cb75e3379819579973b00bdb02efc13cb0083fd3
Tested on the 7TV both V2 and V3, seems to be working just fine

Regarding jChat I left comment https://github.com/tmarenko/twitch_chat_antiban/issues/8#issuecomment-1474876355

**Comment by tmarenko** [comment]
Well that's unfortunate 

![image](https://user-images.githubusercontent.com/31345208/226170668-e7a779e5-4350-4195-b9f5-290d3106049d.png)

**Comment by tmarenko** [comment]
Since 1.5 release there were no bug reports so will consider this fixed
